filebeat/tests/system/test_tcp_tls.py

```diff
@@ -260,7 +260,7 @@ def test_tcp_tls_with_a_plain_text_socket(self):
         # The TLS handshake will close the connection, resulting in a broken pipe.
         # no events should be written on disk.
         with assert_raises(IOError):
-            for n in range(0, 100):
+            for n in range(0, 100000):
```

Comment:
Why did you increase this number? How much time does it add to test execution time?

Reply:
Oh, it's outdated. Nevermind.